### DISCUSSION TITLE
YF 6.5 - Email verification problems

### DISCUSSION THREAD

**Discussion by davide-alghi** [YetiForceCompany/YetiForceCRM#17398] [How to] [answered]
Hi all,
I'm moving a working instance of YF 6.5 to another server and different domain.
I moved the first time: the email verification was successful: I entered the verification code received via email.
I had to redo everything because the PHP files were corrupted.
This time the email verification is not successful and stops here, as in the image.
What should I do?

![Schermata del 2024-03-13 15-19-30](https://github.com/YetiForceCompany/YetiForceCRM/assets/48508691/1fc7c206-3767-4c80-a280-85ceb34c8264)

**Comment by paula-w** [comment, marked as answer]
Hi Davide,
Can you send us your [APP ID](https://doc.yetiforce.com/administrator-guides/app-id/#where-to-find-app-id) to registration@yetiforce.com?

**Reply by davide-alghi** [reply]
Hi Paula, how are you.
tnx for quick response. I'll pick it up from old instance and send it to you right away.

**Reply by davide-alghi** [reply]
@paula-w done.
tnx.

**Reply by davide-alghi** [reply]
problem solved. tnx to YetiForce team
